app/cdap/components/GridTable/index.tsx

```diff
@@ -289,6 +367,30 @@ export default function GridTable() {
           </TableBody>
         </Table>
       )}
+      {openDirectivePanel && (
```

Comment:
it looks like you're conditionally rendering this here and inside the directive input component?

Reply:
Removed from directive input component